pyam/core.py

```diff
@@ -37,11 +37,11 @@
     META_IDX,
     YEAR_IDX,
     IAMC_IDX,
-    SORT_IDX,
-    KNOWN_FUNCS
+    SORT_IDX
 )
 from pyam.read_ixmp import read_ix
 from pyam.timeseries import fill_series
+from pyam.aggregate import _aggregate, _aggregate_region, _agg
```

Comment:
Now that these are in their own module, they should not have preceding underscores. that is left for "private" functions that should not be called out of scope of their own module (file).

Reply:
refactor to no-underscore, renamed `_agg` to `groupby` (that is the main purpose and it is clearly distinct from `aggregate`, even if it is not quite the same as `pandas.DataFrame.groupby`) - if you have better ideas...?